build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -19,24 +19,29 @@ public struct VerInfo
             public string Release;
             public string BuildMajor;
             public string BuildMinor;
+            public string CommitCountString;
 
-            public VerInfo(int major, int minor, int patch, string release, string buildMajor, string buildMinor)
+            public VerInfo(int major, int minor, int patch, string release, string buildMajor, string buildMinor, string commitCountString)
             {
                 Major = major;
                 Minor = minor;
                 Patch = patch;
                 Release = release;
                 BuildMajor = buildMajor;
                 BuildMinor = buildMinor;
+                CommitCountString = commitCountString;
             }
 
             public string GenerateMsiVersion()
             {
-                return Version.GenerateMsiVersion(Major, Minor, Patch, Int32.Parse(BuildMajor));
+                return Version.GenerateMsiVersion(Major, Minor, Patch, Int32.Parse(VerRsrcBuildMajor));
             }
 
             public string WithoutSuffix => $"{Major}.{Minor}.{Patch}";
 
+            public string VerRsrcBuildMajor => !string.IsNullOrEmpty(BuildMajor) ? BuildMajor : CommitCountString;
+            public string VerRsrcBuildMinor => !string.IsNullOrEmpty(BuildMinor) ? BuildMinor : "00";
```

Comment:
Can we rename these properties to follow normal .NET property names?  What does "Rsrc" mean?

If we can't rename these properties, comments are really needed in order to make this code readable.